templates/bot/js/command-and-response/README.md

```diff
@@ -49,148 +47,144 @@ The following files provide the business logic for command and response bot. The
 | `src/helloworldCommandHandler.js` | The business logic to handle a command |
 | `src/cardModels.js` | The default Adaptive Card data model |
```

Comment:
We should remove this since no `cardModels.js` for JavaScript project

Reply:
Done